<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monitor_Spk</class>
 <widget class="QGroupBox" name="Monitor_Spk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel[limitStatus= 0] {border: 1px solid #999999; background-color: rgb(218, 218, 218);}
QLabel[limitStatus= -1] {border: 1px solid #999999; background-color: red;}
QLabel[limitStatus=1] {border: 1px solid #999999; background-color: red;}</string>
  </property>
  <property name="title">
   <string/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>397</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-60</string>
   </property>
  </widget>
  <widget class="VolumeWidget" name="vuMeter" native="true">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>289</y>
     <width>21</width>
     <height>121</height>
    </rect>
   </property>
   <property name="MaxValue" stdset="0">
    <number>0</number>
   </property>
   <property name="MinValue" stdset="0">
    <number>-60</number>
   </property>
   <property name="SingleTrack" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="phase">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>613</y>
     <width>86</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 1px solid #8f8f91;
    border-radius: 2px;
	background-color: rgb(0, 170, 0);
}

QPushButton:pressed {
	background-color: rgb(0, 85, 0);
}
QPushButton:checked{
	background-color: rgb(255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>Phase</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="volumeUpper">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>446</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>12</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>62</y>
     <width>86</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(218, 218, 218);</string>
   </property>
   <property name="text">
    <string>Power:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelError">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>88</y>
     <width>86</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(218, 218, 218);
image: url(:/Image/green.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="volumeLower">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>565</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>283</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="InputSource">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>114</y>
     <width>86</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(218, 218, 218);</string>
   </property>
   <property name="text">
    <string>Net#1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPower">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>62</y>
     <width>86</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 1px solid #8f8f91;
    border-radius: 2px;
	background-color: rgb(255, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(0, 85, 0);
}
QPushButton:checked{
	background-color: rgb(0, 170, 0);
}</string>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>88</y>
     <width>86</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(218, 218, 218);</string>
   </property>
   <property name="text">
    <string>Error:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="devicenamelabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>21</y>
     <width>196</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(135, 206, 235);</string>
   </property>
   <property name="text">
    <string>音箱</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>114</y>
     <width>86</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(218, 218, 218);</string>
   </property>
   <property name="text">
    <string>Input:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="mute">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>639</y>
     <width>86</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 1px solid #8f8f91;
    border-radius: 2px;
	background-color: rgb(0, 170, 0);
}

QPushButton:pressed {
	background-color: rgb(0, 85, 0);
}
QPushButton:checked{
	background-color: rgb(255, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>397</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>dBu</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>446</y>
     <width>20</width>
     <height>131</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>566</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>dB</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurr">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>229</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(218, 218, 218);</string>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolt">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>201</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(218, 218, 218);</string>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>201</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(218, 218, 218);</string>
   </property>
   <property name="text">
    <string>Vol(V):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTemp">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>257</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(218, 218, 218);</string>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>257</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(218, 218, 218);</string>
   </property>
   <property name="text">
    <string>Temp(℃):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>229</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(218, 218, 218);</string>
   </property>
   <property name="text">
    <string>Cur(A):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>589</y>
     <width>44</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(218, 218, 218);</string>
   </property>
   <property name="text">
    <string>Vol</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="vuMeterLLabel">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>415</y>
     <width>48</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(218, 218, 218);</string>
   </property>
   <property name="text">
    <string>-100.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="vuMeterRLabel">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>415</y>
     <width>48</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(218, 218, 218);</string>
   </property>
   <property name="text">
    <string>-100.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>415</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;
background-color: rgb(218, 218, 218);</string>
   </property>
   <property name="text">
    <string>Out/Mon</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="volumeValue">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>589</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VolumeWidget</class>
   <extends>QWidget</extends>
   <header>customWidget/volumewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
